db/migrate/20160223155800_create_audits.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAudits < ActiveRecord::Migration
+  def change
+    create_table :audits do |t|
+      t.references :auditable, polymorphic: true, index: true
+      t.references :user, index: true, foreign_key: true
+      t.references :updater, index: true
+      t.references :vote, index: true, foreign_key: true
+      t.json :audited_changes, default: {}, null: false
```

Comment:
Perhaps this should be in it's own migration file - but I rather wipe the Heroku anyway 😄